src/store/modules/util/index.ts

```diff
@@ -0,0 +1,20 @@
+import actions from './actions';
+import getters from './getters';
+import mutations from './mutations';
+import { Module } from 'vuex';
+import RootState from '@/store/RootState';
+import UtilState from './UtilState';
+
+const utilModule: Module<UtilState, RootState> = {
+  namespaced: true,
+  state: {
+    shipmentMethods: [],
+    country: {},
```

Comment:
```suggestion
    countries: {},
```

Reply:
Sir, Improved variable name from country to countries.